ui/core/proto_utils/action_id.ts

```diff
@@ -339,11 +339,13 @@ export class ActionId {
 			case 'Mana Tide Totem':
 			case 'Power Infusion':
 				if (this.tag != -1) {
-					if (this.tag === playerIndex) {
+					if (this.tag === playerIndex || playerIndex == undefined) {
```

Comment:
individual sim playerIndex was undefined, this is simple check to show (self) on self cast